utilities/transactions/transaction_test.cc

```diff
@@ -4559,6 +4566,124 @@ TEST_P(TransactionTest, MemoryLimitTest) {
   delete txn;
 }
 
+// Test WritePreparedTxnDB's IsInSnapshot against different ordering of
+// snapshot, max_committed_seq_, prepared, and commit entries.
+TEST_P(WritePreparedTransactionTest, IsInSnapshotTest) {
```

Comment:
IsInSnapshotTest needs to be called only with WritePrepared write policy, it cannot be part of TransactionTest.
btw, I am not sure this is important enough to discus it at length.